### ISSUE TITLE
r2 metric report values

### ISSUE THREAD

**Issue by moshe-rl** [mljar/mljar-supervised#495]
When using r2 as eval metric for regression task (with 'Explain' mode) the metric values reported in Leaderboard (at README.md file) are multiplied by -1.
For instance, the metric value for some model shown in the Leaderboard is -0.41, while when clicking the model name leads to the detailed results page - and there the value of r2 is 0.41.
I've noticed that when one of R2 metric values in the Leaderboard was > 1 (~ 45), which shouldn't be possible for R2 metric, then I looked in the detailed results and saw that the value is actually ~ -45.

**Comment by pplonski** [comment]
Might be a bug. Thanks for reporting. Could you please provide code to reproduce?

**Comment by moshe-rl** [comment]
`
```
import numpy as np
import pandas as pd
from supervised.automl import AutoML
import tempfile
import os

# create random input data
data = np.random.randn(1000, 11)

# convert to dataframe
data_df = pd.DataFrame(data)
data_df.columns = [f'f{idx}' for idx in range(data.shape[1]-1)] + ['y']

# separate x and y
y_vec = data_df.pop('y').values
x_df = data_df

# train model
model = AutoML(results_path=os.path.join(tempfile.gettempdir(), 'aml_results__2'), mode='Explain', eval_metric='r2')
model.fit(x_df, y_vec)
```
`

**Comment by pplonski** [comment]
@MaciekEO could you try to reproduce this bug?

**Comment by MaciekEO** [comment]
@moshe-rl I recreated your code and there are some negative values, but when I changed data in df I've got good results.
You should check this code on different data.

**Comment by pplonski** [comment]
Thanks @MaciekEO for checking, @moshe-rl could you please provide data to reproduce the issue? If you can't post it on GitHub maybe you can send it by email to contact@mljar.com?

The R2 metric can be negative - see the R2 [docs](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html) is sklearn package.

**Comment by moshe-rl** [comment]
Hey @pplonski @MaciekEO,
The point was that the R2 values I saw **at the leader-board** in the main result page **was X** for model Y (for instance), but **when I click on the name** of model Y (in the leader-board) to get to the **detailed result page**- I see **there that the R2 value is -X**.
This behavior leads to strange results - for some model the **actual** R2 value (which present in the **detailed model result page**) was -45 (which is certainly legitimate), but **in the leader-board** I saw value of +45 - which is illegal for R2.

**Comment by pplonski** [comment]
Thanks @moshe-rl for the reply. The point is that we can't reproduce the issue. With the code that you provided (with random data), there are negative R2 values but they are negative in the console, leaderboard, and model description, so I assume they are valid (R2 can be negative). We need some code&data example with R2 wrong values (for example > 1) so we can find the bug and fix it.
